docker/bragi/Dockerfile

```
@@ -39,37 +28,27 @@ COPY ./benches ./benches
 
 RUN --mount=type=cache,target=/usr/local/cargo/registry \
     --mount=type=cache,target=/home/mimirsbrunn/target  \
-    cargo build --release --bin bragi --locked --features db-storage
+    cargo build --profile production --bin bragi --locked --features db-storage
 
 # Extract binary from build cache
 RUN mkdir bin
 RUN --mount=type=cache,target=/home/mimirsbrunn/target \
-    cp target/release/bragi bin/
+    cp target/production/bragi bin/
 
 
-ARG DEBIAN_VERSION
+# Target image
+# ============
 
-FROM debian:${DEBIAN_VERSION}-slim
+FROM debian:buster-slim
 
 WORKDIR /srv
 
 ENV DEBIAN_FRONTEND noninteractive
 
-ARG DEBIAN_VERSION
-
-RUN if [ "${DEBIAN_VERSION}" = "buster" ]; then \
-  apt-get update \
-    && apt-get install -y curl libcurl4 sqlite3 \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*; \
-elif [ "${DEBIAN_VERSION}" = "stretch" ]; then \
-  apt-get update \
-    && apt-get install -y curl libcurl3 sqlite3 \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*; \
-else \
-  echo "Unsupported debian version '$DEBIAN_VERSION'"; \
-fi
+RUN apt-get update
+RUN apt-get install -y libcurl4 sqlite3
```

```suggestion
RUN apt-get install -y curl libcurl4 sqlite3
```
we do need curl in the docker for implementing an healthcheck